### PR TITLE
[15.0][ADD] purchase_fully_received

### PR DESCRIPTION
**Pull request by AaronHForgeFlow**
Split from https://github.com/OCA/purchase-workflow/pull/1647

because of this comment: https://github.com/OCA/sale-workflow/pull/2306#pullrequestreview-1218920523

cc @ForgeFlow

### CODE REVIEW

File: purchase_fully_received/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Copyright 2022 ForgeFlow S.L.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html)
+
+{
+    "name": "Purchase Fully Received",
+    "summary": "Useful filters in Purchases to know the actual status of shipments "
```
Comment (MiquelRForgeFlow) [inline purchase_fully_received/__manifest__.py]:
```suggestion
    "summary": "Useful filters in Purchases to know the actual status of shipments."
```

**Review by LoisRForgeFlow — APPROVED** [review]
Code review LGTM

**Review by JasminSForgeFlow — APPROVED** [review]

**Review by HviorForgeFlow — APPROVED** [review]
LGTM

**Pull request closed without merging**